### PR TITLE
Initial notes pages

### LINKED ISSUE #1: Add organizer notes from 1/7/20 discussion
Add notes from the discussion between @lortza and @hayesmp regarding:
* format
* topics
* food options
* notes for next meetup

### PR DESCRIPTION
**Pull request by lortza**
Closes #1

This is WIP and we can converse via comments during the process.

I intended for this to be a good spot for markdown files for these purposes:
* organizers to brainstorm together and save ideas in a shared space
* keep myself organized with topics i want to be sure to bring up on a certain date 
* any follow-up notes for a specific meetup can be tracked or noted as issues which we can assign to ourselves 
* us to post code challenges or any meeting-related code snippets we want to discuss
* post Welcome info (if we have a screen to project it on)

You can click thorough the documentation on this branch here where it is in its full markdown glory: https://github.com/wwcodeatx/ruby_tuesday/tree/1_add_notes

### CODE REVIEW

File: organizer_notes/food.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Food Resources
+
+## Genre Ideas
+Indian, greek/middle-eastern, pizza, thai, fajitas, Chipolte, chinese
+
+## Restaurants We Like
+(easy, reliable, good options)
+
+* wip
```
Comment (lortza) [inline organizer_notes/food.md]: @hayesmp can you add in the food vendors that you've used before and like?

File: welcome.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Please grab some food and make yourself comfortable.
+
+## Getting Connected
+* Slack: wwc-atx.slack.com
+* Twitter: @wwcodeatx
+* Web: womenwhocode.com/austin: Coding Resources, Job Board, Meetups, and MOAR!
+
+## Your Organizers
+* Miriam Hayes: contact info
+* Anne Richardson:  contact info
```
Comment (lortza) [inline welcome.md]: We should put some kind of contact info here. I'm not sure how to do this anymore! Generally i prefer for people to DM me via slack, but if they're not members, how do they do that?

Also, do we have an easy way to add people to the slack group?

File: organizer_notes/food.md (hunk 1 above)
Comment (hayesmp) [reply organizer_notes/food.md]: We use Foodee. The only restaurant I am aware of is Taj Palace for Indian.

File: welcome.md (hunk 2 above)
Comment (hayesmp) [reply welcome.md]: People sometimes post in the Meetup.

`invite_people name@internet.com` 

We can also message cc or sara or post in the organizers channel.

Comment (lortza) [reply welcome.md]: Is `invite_people name@internet.com` a slack bot integration? How do we use it?

Comment (hayesmp) [reply welcome.md]: Yes. Slackbot. 
type `/invite_people miriam@internet.org` into the slack textbox in any channel.

**Pull request merged**